.gitignore

```diff
@@ -51,3 +51,4 @@ bower.json
 node_modules/
 
 test/*
+Gemfile.lock
```

Comment:
Why you added this file to the .gitignore? Don't you need to control the version of your gems?
Please remove this from gitignore. :)))